region/maridia/inner-pink.json

```diff
@@ -1012,7 +1035,8 @@
                   "canSwim",
                   {"canComeInCharged": {
                     "fromNode": 2,
-                    "framesRemaining": 30
+                    "framesRemaining": 30,
+                    "shinesparkFrames": 45
```

Comment:
You don't need to go all the way up here. You can do a diagonal left spark for 20 frames.

Reply:
With 30 framesRemaining I imagined it was balanced towards short time rather than saving health but if you say 30 frames is enough for the diagonal I guess that's the one intended.

Reply:
I tested it before writing the comment and it's even doable in 20 frames. You really don't need to do far off the door because a diagonal shinespark doesn't go at 45 degrees.